src/sentry/flags/models.py

```diff
@@ -0,0 +1,65 @@
+from enum import Enum
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model, region_silo_model, sane_repr
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+class ActionEnum(Enum):
+    CREATED = 0
+    DELETED = 1
+    UPDATED = 2
+
+
+ACTION_MAP = {
+    "created": ActionEnum.CREATED.value,
+    "deleted": ActionEnum.DELETED.value,
+    "updated": ActionEnum.UPDATED.value,
```

Comment:
these values differ from the blueprint, right?

Reply:
blueprint specifies `created` or `modified`

Reply:
Good catch.  I'll update the blueprint to reflect.